### DISCUSSION TITLE
Tracking on yolov8 is not working when i have stream = True

### DISCUSSION THREAD

**Discussion by scopeturned** [ultralytics/ultralytics#15648] [Q&A] [answered]
I recently started working with yolo on an object tracking project but got stuck when i've tried working with the track option that it provides. When i use stream = True attribute on the track option and try to run it the window opens for a split second and then closes. I want to also include stream = True in my code since if not then all my ram will be eaten up by the track option.

I am using the default weights (yolov8n.pt).

My code:

from ultralytics import YOLO

model = YOLO("yolov8n.pt")

model.track(source = 0, show=True, stream = True)

**Comment by glenn-jocher** [comment, marked as answer]
@scopeturned thank you for reaching out. The issue you're experiencing with `stream=True` might be due to a bug or an environment-specific problem. Please ensure you are using the latest version of the Ultralytics package. If the issue persists, consider opening an issue on our GitHub page with detailed information about your environment and any error messages. This will help us investigate further.

**Reply by scopeturned** [reply]
@glenn-jocher thank you for your response. I will update all my libraries and reach up to you with more details if the issue persists.

**Reply by glenn-jocher** [reply]
You're welcome! Please do update your libraries and let us know if the issue persists. We're here to help.

**Reply by scopeturned** [reply]
I updated them and now it works. Thanks for the help!

**Reply by glenn-jocher** [reply]
Great to hear that it's working now! If you have any more questions, feel free to ask.
